crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -103,6 +104,7 @@ impl<'a> LinkStage<'a> {
 
     self.determine_module_exports_kind();
     self.wrap_modules();
+    self.generate_lazy_export();
```

Comment:
I don't find step 3 related in our codebase, so I add another phase, so phase should not have high overhead.